src/main/resources/META-INF/plugin.xml

```diff
@@ -206,9 +206,7 @@
         <action id="GitMachete.OpenMacheteTabAction"
                 class="com.virtuslab.gitmachete.frontend.actions.vcsmenu.OpenMacheteTabAction"
                 icon="com.virtuslab.gitmachete.frontend.icons.MacheteIcons.MACHETE_FILE">
-            <!-- In future IntelliJ releases this group-id will probably be `Git.MainMenu`
-                 We can consider `VcsGroups` that seems to be more stable and place this action in VCS menu -->
-            <add-to-group group-id="GitRepositoryActions" anchor="last"/>
+            <add-to-group group-id="VcsGroups" anchor="last"/>
```

Comment:
Pls add a comment about where this `<add-group>` puts our action... I assume it's _Git menu_ as in, here:
![image](https://user-images.githubusercontent.com/3383210/192769318-47fba260-60bf-4c63-9351-2b73d83f5c4a.png)
?

Reply:
<img width="486" alt="image" src="https://user-images.githubusercontent.com/19799111/192769649-4acdc09b-0d01-47da-9c5a-5efc96e418b0.png">